src/style.scss

```diff
@@ -8,12 +8,30 @@ $prefix: sixa;
 .wp-block-#{$prefix}-faq {
 
 	&__title {
+		display: flex;
+		align-items: center;
+		justify-content: space-between;
+
 		list-style: none;
 		cursor: pointer;
 
 		&:focus {
 			outline: 0;
 		}
+
+		&::after {
+			content: "";
+			width: 0;
+			height: 0;
+			border-left: 8px solid transparent;
+			border-right: 8px solid transparent;
+			border-top: 8px solid var(--wp--custom--color--paragraph, #000);
```

Comment:
Wouldn’t it be better to use `currentColor` to inherit the border color?

Reply:
> Wouldn’t it be better to use `currentColor` to inherit the border color?

I am not familiar with this feature 😄 . If `currentColor` works and is easier than the CSS variable, I'm all in favour of replacing it.

Reply:
The border color will inherit the after selector color from the FAQ title text color. It would be helpful to add the color setting to the block for the title. Then the icon will have the same title color.

Reply:
Thank you, @gooklani . I have updated the CSS according to your suggestion.